lib/compress.js

```diff
@@ -2292,7 +2266,18 @@ merge(Compressor.prototype, {
                 }
             }
         }
-        return self.evaluate(compressor)[0];
+        function is_iife_call(node) {
```

Comment:
to improve readability please move is_iife_call below the return at the end of the function.